packages/forma-36-website/src/pages/guidelines/copy/date-and-time/index.mdx

```diff
@@ -0,0 +1,239 @@
+---
+title: 'Date and time'
+---
+
+import {
+  Table,
+  TableHead,
+  TableBody,
+  TableCell,
+  TableRow,
+  Tooltip,
+} from '@contentful/forma-36-react-components';
+
+# Date and time
+
+## Absolute vs relative
+
+Consider the context when having to choose between absolute and relative date and time.
+
+<Table className="f36-margin-bottom--xl" style={{ width: '100%' }}>
+  <TableHead>
+    <TableRow>
+      <TableCell style={{ width: '25%' }}>Type</TableCell>
+      <TableCell style={{ width: '25%' }}>Example</TableCell>
+      <TableCell style={{ width: '50%' }}></TableCell>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    <TableRow>
+      <TableCell>Absolute</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>13 Aug 2019 at 10:00 am</TableCell>
+      <TableCell>
+        When is an entry scheduled to be published?
+        <br />
+        Relative time wouldn't be specific enough.
+      </TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Relative</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        <Tooltip content="13 Aug 2019 at 10:00 am">3 days ago</Tooltip>
```

Comment:
@m10l @wichniowski it would be cool to display actual "3 days ago" time here instead of this "13 Aug 2019 at 10:00 am" placeholder :)